sonar-text-plugin/src/test/java/org/sonar/plugins/secrets/checks/AwsCheckTest.java

```diff
@@ -31,6 +31,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.sonar.plugins.common.TestUtils.analyze;
 
+
```

Comment:
Do we need a ticket to add code style checking or spotless plugin?

Reply:
Spotless would be good. Let's create a ticket for it.